gradle/testing/failed-tests-at-end.gradle

```
@@ -25,11 +25,12 @@ allprojects {
   tasks.withType(Test) { Test task ->
     afterTest { desc, result ->
       if (result.resultType == TestResult.ResultType.FAILURE) {
+        def testOrClassName = desc.name == "classMethod" ? desc.className : "${desc.className}.${desc.name}"
         failedTests << [
             "name": "${desc.className}.${desc.name}",
```

How about also changing this?
```suggestion
            "name": "$testOrClassName",
```

(I spotted this only because in https://github.com/apache/solr/actions/runs/9274510162/job/25517863554?pr=2483 it says _"... ERROR: The following test(s) have failed: ... org.apache.solr.ltr.feature.TestUserTermScoreWithQ.classMethod (:solr:modules:ltr) ..."_ currently.)

---

Hmm, its still probably good to have a pretty clear indication that it was a classMethod, right? I think it would be a bit confusing otherwise to see that both `org.apache.solr.ltr.feature.TestUserTermScoreWithQ.testUserTermScoreWithQ` and `org.apache.solr.ltr.feature.TestUserTermScoreWithQ` failed

---

It just happened so that in the GHA check (https://github.com/apache/solr/actions/runs/9274510162/job/25517863554) for this PR, a couple of tests failed - and one could see how the failures are reported:

* lines 479-478
```
> Task :solr:modules:ltr:test

org.apache.solr.ltr.feature.TestUserTermScoreWithQ > testUserTermScoreWithQ FAILED
    java.lang.Exception: Test abandoned because suite timeout was reached.
        at __randomizedtesting.SeedInfo.seed([1B9234167BEFFA61]:0)

org.apache.solr.ltr.feature.TestUserTermScoreWithQ > classMethod FAILED
    java.lang.Exception: Suite timeout exceeded (>= 600000 msec).
        at __randomizedtesting.SeedInfo.seed([1B9234167BEFFA61]:0)
```

* line 1017
```
2> NOTE: reproduce with: gradlew test --tests TestUserTermScoreWithQ.testUserTermScoreWithQ -Dtests.seed=1B9234167BEFFA61 -Dtests.locale=ar-BH -Dtests.timezone=WET -Dtests.asserts=true -Dtests.file.encoding=UTF-8
```

* line 1404
```
2> NOTE: reproduce with: gradlew test --tests TestUserTermScoreWithQ -Dtests.seed=1B9234167BEFFA61 -Dtests.locale=ar-BH -Dtests.timezone=WET -Dtests.asserts=true -Dtests.file.encoding=UTF-8
```

* lines 1413-1420
```
ERROR: The following test(s) have failed:
  - org.apache.solr.ltr.feature.TestUserTermScoreWithQ.testUserTermScoreWithQ (:solr:modules:ltr)
    Test output: /tmp/src/solr/solr/modules/ltr/build/test-results/test/outputs/OUTPUT-org.apache.solr.ltr.feature.TestUserTermScoreWithQ.txt
    Reproduce with: gradlew :solr:modules:ltr:test --tests "org.apache.solr.ltr.feature.TestUserTermScoreWithQ.testUserTermScoreWithQ" -Ptests.jvms=96 "-Ptests.jvmargs=-XX:TieredStopAtLevel=1 -XX:+UseParallelGC -XX:ActiveProcessorCount=1 -XX:ReservedCodeCacheSize=120m" -Ptests.seed=1B9234167BEFFA61 -Ptests.timeoutSuite=600000! -Ptests.file.encoding=US-ASCII

  - org.apache.solr.ltr.feature.TestUserTermScoreWithQ.classMethod (:solr:modules:ltr)
    Test output: /tmp/src/solr/solr/modules/ltr/build/test-results/test/outputs/OUTPUT-org.apache.solr.ltr.feature.TestUserTermScoreWithQ.txt
    Reproduce with: gradlew :solr:modules:ltr:test --tests "org.apache.solr.ltr.feature.TestUserTermScoreWithQ" -Ptests.jvms=96 "-Ptests.jvmargs=-XX:TieredStopAtLevel=1 -XX:+UseParallelGC -XX:ActiveProcessorCount=1 -XX:ReservedCodeCacheSize=120m" -Ptests.seed=1B9234167BEFFA61 -Ptests.timeoutSuite=600000! -Ptests.file.encoding=US-ASCII
```

Note that this PR only adjusts the line 1420 from the last block, - it now becomes consistent with line 1404 which already knew not to append "classMethod" to the class name.